src/index.spec.ts

```diff
@@ -1,12 +1,182 @@
+import cases from 'jest-in-case';
+import fetch from 'node-fetch';
+import { mocked } from 'ts-jest/utils';
 import {
   isInteger,
   toLowerCase,
   removeDuplicatesFromArray,
   createRandomProduct,
   getStarWarsPlanets,
-  createProduct 
+  createProduct,
 } from './index';
 
-test('Testing time!', () => {
-  console.log('welcome');
+jest.mock('node-fetch', () => jest.fn());
+
+describe('isInteger', () => {
+  cases(
+    'should return',
+    (opts) => {
+      expect(isInteger(opts.actual)).toBe(opts.expected);
+    },
+    [
+      { name: 'true evaluating a number', actual: 123, expected: true },
+      {
+        name: 'true evaluating a negative number',
+        actual: -123,
+        expected: true,
+      },
+      { name: 'false evaluating a string', actual: '1', expected: false },
+      { name: 'false evaluating a float', actual: 1.25, expected: false },
+    ],
+  );
+});
+
+describe('toLowerCase', () => {
+  cases(
+    `should return`,
+    (opts) => {
+      expect(toLowerCase(opts.actual)).toEqual(opts.expected);
+    },
+    [
+      {
+        name: 'prueba when PrueBa is given',
+        actual: 'PrueBa',
+        expected: 'prueba',
+      },
+      {
+        name: 'the same string when a lowercase string is given',
+        actual: 'lowercase',
+        expected: 'lowercase',
+      },
+      {
+        name: 'return error when passing null or an empty string',
+        actual: null,
+        expected: 'Please provide a string',
+      },
+    ],
+  );
+});
+
+describe('removeDuplicatesFromArray', () => {
+  cases(
+    `should return`,
+    (opts) => {
+      expect(removeDuplicatesFromArray(opts.actual)).toStrictEqual(
+        opts.expected,
+      );
+    },
+    [
+      {
+        name: '[1,2,3] when [1,2,2,3,1] is evaluated',
+        actual: [1, 2, 2, 3, 1],
+        expected: [1, 2, 3],
+      },
+      {
+        name: `['1','2','3'] when ['1','2','2','3','1'] is evaluated`,
```

Comment:
use only back tips when you are using the values for variables inside, in this case, you must use double quotes